### PR TITLE
Remove unnecessary judgment

### PR DESCRIPTION
**Pull request by ZP-AlwaysWin**
**What type of PR is this?**
Remove unnecessary judgment
/kind cleanup

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

```
aL map[string]string

so nil len(aL) ==0 

change:
 if aL == nil || len(aL) == 0 ==>if len(aL) == 0 
```

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

### CODE REVIEW

File: pkg/scheduler/algorithm/predicates/utils.go [hunk 1]
```diff
@@ -67,7 +67,7 @@ func FilterPodsByNamespace(pods []*v1.Pod, ns string) []*v1.Pod {
 
 // CreateSelectorFromLabels is used to define a selector that corresponds to the keys in a map.
 func CreateSelectorFromLabels(aL map[string]string) labels.Selector {
-	if aL == nil || len(aL) == 0 {
+	if len(aL) == 0 {
```
Comment (clarklee92) [inline pkg/scheduler/algorithm/predicates/utils.go]: `nil` can not be pasted to len()(Although len(nil)=0, explicit expression is better), I think the judgement of map `nil` or `empty` is necessary. 😝

**Comment by alculquicondor** [comment]
Please add the release note

/priority backlog
/lgtm

**Comment by ZP-AlwaysWin** [comment]
/assign @liggitt
PTAL

**Pull request closed without merging**